redirect.py

```diff
@@ -41,8 +41,20 @@ def get(self):
           self.response.write(json.dumps(result))
       elif self.request.get('site') == 'comments':
           urlfetch.set_default_fetch_deadline(10)
+          startIndex = self.request.get('start')
+
+          try:
+            startIndex = int(startIndex)
+          except:
+            startIndex = 0
+
+          maxResults = self.request.get('max')
+          try:
+            maxResults = int(maxResults)
+          except:
+            maxResults = 0
           self.response.headers.add_header("Access-Control-Allow-Origin", "*")
-          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId')).execute()
+          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId'), maxResults=max, startIndex=startIndex).execute()
```

Comment:
Yup. However we still probably want to know what to default to.

It's 100 for issues (I think) but that isn't encoded in redirect.py yet. I should fix that so that people can't accidentally ask for too many issues.

It is unlikely to be the same number for comments because the actual number that can be fetched depends heavily on size and characteristics of how the data is stored in the tool. I'd manually try 100, 200, 500 a bunch of times and see what happens (this *does* mean you need to find issues with more than 500 comments...).